tutorials/docs-01-contributing-guide/index.qmd

```diff
@@ -2,17 +2,28 @@
 title: Contributing
 ---
 
-Turing is an open-source project. If you feel that you have relevant skills and are interested in contributing, please get in touch. You can contribute by opening issues on GitHub, implementing things yourself, and making a pull request. We would also appreciate example models written using Turing.
+Turing is an open-source project and is [hosted on GitHub](https://github.com/TuringLang).
+We welcome contributions from the community in all forms large or small: bug reports, feature implementations, code contributions, or improvements to documentation or infrastructure are all extremely valuable.
+We would also very much appreciate examples of models written using Turing.
 
-Turing has a [style guide](#style-guide). Reviewing it before making a pull request is not strictly necessary, but you may be asked to change portions of your code to conform with the style guide before it is merged.
+### How to get involved
 
-### What Can I Do?
+Our outstanding issues are tabulated on our [issue tracker](https://github.com/TuringLang/Turing.jl/issues).
+Closing one of these may involve implementing new features, fixing bugs, or writing example models.
 
-Look at the [issues](https://github.com/TuringLang/Turing.jl/issues) page for an outstanding issue. For instance, you could implement new features, fix bugs or write example models.
+You can also join the `#turing` channel on the [Julia Slack](https://julialang.org/slack/) and say hello!
 
-### Git Workflow
+If you are new to open-source software, please see [GitHub's introduction](https://guides.github.com/introduction/flow/) or [Julia's contribution guide](https://github.com/JuliaLang/julia/blob/master/CONTRIBUTING.md) on using version control for collaboration.
 
-For more information on how the Git workflow typically functions, please see the [GitHub's introduction](https://guides.github.com/introduction/flow/) or [Julia's contribution guide](https://github.com/JuliaLang/julia/blob/master/CONTRIBUTING.md).
+### Documentation
+
+Each of the packages in the Turing ecosystem (see [Libraries](/library)) has its own documentation, which is typically found in the `docs` folder of the corresponding package.
+For example, the source code for DynamicPPL's documentation can be found in [its repository](https://github.com/TuringLang/DynamicPPL.jl).
+
+The documentation for Turing.jl itself consists of the tutorials that you see on this website, and is built from the separate [`docs` repository](https://github.com/TuringLang/docs).
+None of the documentation is generated from the Turing.jl repository; in particular, the API that Turing exports does not currently form part of the documentation.
+
+Other sections of the website (anything that isn't a package, or a tutorial)—for example, the list of libraries—is built from the [`turinglang.github.io` repository](https://github.com/TuringLang/turinglang.github.io).

```

Comment:
```suggestion
Other sections of the website (anything that isn't a package, or a tutorial) — for example, the list of libraries — is built from the [`turinglang.github.io` repository](https://github.com/TuringLang/turinglang.github.io).
```
Not sure how Quarto does hyphens and whitespace, but might some space here be good?

Reply:
This is an em dash. Classically, em dashes aren't surrounded by spaces.

However, [Oxford's style guide (p 13)](https://www.ox.ac.uk/sites/files/oxford/media_wysiwyg/University%20of%20Oxford%20Style%20Guide.pdf) suggests using en dashes with spaces for the same purpose 😅